contracts/atomic-asset-typescript/lib/utils.ts

```diff
@@ -0,0 +1,30 @@
+export function getOr<T = any>(value: T, defaultVal: T): T {
+    if (value) {
+        return value;
+    }
+
+    return defaultVal;
+}
+
+export function get(value: any) {
+    if (!value) {
+        throw new ContractError(`Undefined value!`);
```

Comment:
not the most informative error in the world :)

Reply:
Hmm, yep it should be something like "Null Pointer Exception", it should never happen.
I lookup it is only used to get `SmartWeave.caller` which is nullable, so change it to `getCaller()`